src/js/plugin/sdk/pipeline/PipelineStore.js

```diff
@@ -0,0 +1,23 @@
+import PipelineModel from "./PipelineModel";
+
+class PipelineStore {
+
+  constructor() {
+    Object.defineProperty(this, "pipeline", {value: new PipelineModel()});
```

Comment:
🎨  I'd rename the value to "pipelines" as we're allowing users to register operators for multiple different pipelines.